### PR TITLE
fix: updating validation in QueueProcessor

### PR DESCRIPTION
**Pull request by krackerman**
to allow undefined 'queueUrl'.

### CODE REVIEW

File: src/module/sqs/send-message-batch.js [hunk 1]
```diff
@@ -47,12 +49,22 @@ const sendBatch = async (sqsBatch, queueUrl, {
...
+    delaySeconds: Joi.number().integer().optional()
+  }));
+  const msgs = get(opts, 'msgs');
+  const queueUrl = get(opts, 'queueUrl');
+  const batchSize = get(opts, 'batchSize', 10);
+  const maxRetries = get(opts, 'maxRetries', 10);
+  const backoffFunction = get(opts, 'backoffFunction', (count) => 30 * (count ** 2));
+  const delaySeconds = get(opts, 'delaySeconds', null);
+
   assert(batchSize <= 10, 'AWS sqs:sendMessageBatch restriction');
```
Comment (krackerman) [inline src/module/sqs/send-message-batch.js]: Should we validate this with the joi schema?

File: README.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ Get the service from the underlying `aws-sdk` without initializing it. Possible
 Updates the global aws config of the underlying `aws-sdk` via `AWS.config.update`.
 In most cases this should not be necessary to use.
 
-#### sqs.sendMessageBatch(msgs: Array, queueUrl: String, options: Object = {})
```
Comment (simlu) [inline README.md]: fix

Comment (simlu) [reply README.md]:
```suggestion
#### sqs.sendMessageBatch(msgs: Array, queueUrl: String /* ... other options ... /* })
```

File: src/module/sqs/send-message-batch.js [hunk 3]
```diff
@@ -47,12 +49,22 @@ const sendBatch = async (sqsBatch, queueUrl, {
...
 
-module.exports = ({ call, getService, logger }) => async (msgs, queueUrl, {
-  batchSize = 10,
-  maxRetries = 10,
-  backoffFunction = (count) => 30 * (count ** 2),
-  delaySeconds = null
-} = {}) => {
+module.exports = ({ call, getService, logger }) => async (opts) => {
+  Joi.assert(opts, Joi.object().keys({
+    msgs: Joi.array(),
```
Comment (simlu) [inline src/module/sqs/send-message-batch.js]:
```suggestion
    messages: Joi.array(),
```

**Pull request merged**